projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-case-details-header/query-case-details-header.component.html

```diff
@@ -0,0 +1,15 @@
+<div *ngIf="caseView">
+  <div class="govuk-body-l govuk-!-margin-bottom-0">{{ caseView.case_id | ccdCaseReference }}</div>
+
+  <ng-container>
+    <ng-container *ngIf="caseView?.state?.title_display; else caseNameMissing">
+      <ccd-markdown [content]="caseView.state.title_display"></ccd-markdown>
+    </ng-container>
+
+    <ng-template #caseNameMissing>
+      <div class="govuk-body-l govuk-!-margin-bottom-0">
+        Case name missing
```

Comment:
@MateiHMCTS - this needs to be piped through `rpxTranslate` for Welsh language support